src/components/Arjuntool/Arjuntool.tsx

```diff
@@ -83,6 +86,11 @@ export function Arjuntool() {
 
         const args = ["-u", values.url];
 
+        // Conditional. If the user has specified stability, add the --stable option to the command.
+        if (values.stability == true) {
```

Comment:
Hi @michaeljpigott,

This can be simplified. When conducting a conditional statement such as an if statement the argument inside () is used to check its truthiness of the expression.

Given that if the values.stability contains an argument it will be true the == true is redundant and can be removed.

- [ ] Remove == true and test.

Thank you!

Reply:
Tested and confirmed that it works without the == true.